examples/tonic-key-value-store/src/main.rs

```diff
@@ -188,10 +216,36 @@ impl key_value_store_server::KeyValueStore for ServerImpl {
         let SetRequest { key, value } = request.into_inner();
         let value = Bytes::from(value);
 
+        self.tx
+            .send(SubscribeReply { key: key.clone() })
+            .expect("failed to send");
+
         self.db.write().unwrap().insert(key, value);
 
         Ok(Response::new(SetReply {}))
     }
+
+    type SubscribeStream =
+        Pin<Box<dyn Stream<Item = Result<SubscribeReply, Status>> + Send + Sync + 'static>>;
+
+    async fn subscribe(
+        &self,
+        request: Request<SubscribeRequest>,
+    ) -> Result<Response<Self::SubscribeStream>, Status> {
+        let SubscribeRequest {} = request.into_inner();
```

Comment:
What is this syntax? Are you matching on an empty struct huh

Reply:
Mostly to get a compile error if/when we add more fields. Not strictly necessary for this example but might help users who copy+paste this.